docker-compose.test.yml

```diff
@@ -0,0 +1,3 @@
+sut:
```

Comment:
What is "sut" and when do we use this file? I guess it somehow finds use in the testing of the image on docker hub?